docs/user_guide/labels_and_annotations_in_yunikorn.md

```diff
@@ -25,19 +25,21 @@ under the License.
-|-----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `applicationId`                   | Associates this pod with an application.                                                                                                                |
-| `queue`                           | Selects the YuniKorn queue this application should be scheduled in. This may be ignored if a placement policy is in effect.                             |
-| `spark-app-selector`              | Alternative method of specifying `applicationId` used by Spark Operator if the label `applicationId` and annotation `yunikorn.apache.org/app-id` unset. |
+Label values should comply with [Kubernetes Syntax and character set](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set).
+
+| Name                              | Description                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+|-----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `applicationId`                   | Associates this pod with an application.                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `queue`                           | Selects the YuniKorn queue this application should be scheduled in. Queue name should comply with [Kubernetes Syntax and character set](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) and also with [Partition and Queue Configuration](queue_config#queues). If Queue name doesn't comply, Annotations could be used as described below. This may be ignored if a placement policy is in effect. |
```

Comment:
Note: The order has been changed in [[YUNIKORN-2504] Support canonical labels and align metadata retrieving order in shim ](https://issues.apache.org/jira/browse/YUNIKORN-2504) (Annotation queue prior to legacy label queue) .  It't not related to this PR. I will create another Jira to change it once the 1.6.0 related Jira under [YUNIKORN-2501](https://issues.apache.org/jira/browse/YUNIKORN-2501) are done.